### PR TITLE
[hi] Localize /content/hi/docs/reference/glossary/kubelet.md

### PR DESCRIPTION
**Pull request by vaibhav2107**
Fixes: #37515

### CODE REVIEW

**Comment by vaibhav2107** [comment]
Hi @anubha-v-ardhan @divya-mohan0209 ,
Could you please have a look

**Comment by vaibhav2107** [comment]
Hi @bishal7679,
Could you please have a look.

**Review by Babapool — COMMENTED** [review]
Hope you find these suggestions useful. Thanks.

**Comment by vaibhav2107** [comment]
Thanks @Babapool for reviewing and suggesting the changes

**Comment by vaibhav2107** [comment]
@Babapool I think the suggestions are good. I am ready to commit them if some official member could approve them

**Comment by Babapool** [comment]
Hi @vaibhav2107, just a friendly reminder are you still working on this PR? Thanks

**Comment by bishal7679** [comment]
@vaibhav2107 Gentle reminder!

**Comment by vaibhav2107** [comment]
@Babapool PTAL
I have updated your given suggestions

File: content/hi/docs/reference/glossary/kubelet.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- fundamental
+---
+एक एजेंट जो क्लस्टर में प्रत्येक {{< glossary_tooltip text="नोड" term_id="node" >}} पर चलता है। यह सुनिश्चित करता है कि {{<glossary_tooltip text="कंटेनर" term_id="container" >}} एक {{<glossary_tooltip text="पॉड" term_id="pod" >}} में चल रहे हैं।
+
+<!--more-->
+
+क्यूबलेट को विभिन्न तंत्रों के माध्यम से एक पॉडस्पेक्स (PodSpec) समूह प्राप्त होता है और यह सुनिश्चित करता हैं कि इन पॉडस्पेक्स में वर्णित कंटेनर चल रहे हैं और स्वस्थ हैं। क्यूबलेट उन कंटेनरों का प्रबंधन नहीं करता है जो कुबेरनेट्स द्वारा नहीं बनाए गए थे।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/kubelet.md:17]: Nit.
```suggestion
क्यूबलेट को विभिन्न तंत्रों के माध्यम से पॉडस्पेक्स (PodSpec) का एक समूह प्राप्त होता है और यह सुनिश्चित करता हैं कि इन पॉडस्पेक्स में वर्णित कंटेनर चल रहे हैं और स्वस्थ हैं। क्यूबलेट उन कंटेनरों का प्रबंधन नहीं करता है जो कुबेरनेट्स द्वारा नहीं बनाए गए थे।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Comment by Babapool** [comment]
Hi @vaibhav2107, just a gentle ping, since there has been no updates on this PR for a long time are you still working on this PR? Thanks

**Comment by bishal7679** [comment]
!!Gentle Reminder!!
Hey @vaibhav2107 Would you like to have a look on this PR since it's been 4 month along with no update!
Feel free to comment here without any hesitation
Hope you understand
Thank you!

**Comment by vaibhav2107** [comment]
Hi @Babapool @bishal7679 
I have updated the PR for the change. Could you please have a look now

**Pull request merged**